src/components/userFeedback/UserFeedback.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { extend } from 'underscore';
+
+export interface IUserFeedbackProps {
+  feedbackText: string;
+  state: string;
+  extraClasses?: string;
+  displayOnShow?: string;
+}
+
+export interface IUserFeedbackStyle {
+  displayClass: string;
```

Comment:
`displayClass` is required but there's a default value in `getDefaultUserFeedbackStyle` ?

Is there a way that we could merge extraClasses and displayClass? It looks like the same thing to me